apps/prairielearn/src/sprocs/instance_questions_grade.sql

```diff
@@ -17,10 +17,23 @@ BEGIN
 
     IF NOT instance_question_open THEN
         -- this shouldn't happen, so log an error
+        --
+        -- TODO: We should actually work to prevent this from happening
+        -- farther upstream, and avoid recording an issue here.
         PERFORM issues_insert_for_variant(
-            v.id, 'Submission when instance question is closed', '', false,
-            false, jsonb_build_object('grading_job_id', grading_job_id),
-            '{}'::jsonb, instance_questions_grade.authn_user_id)
+            v.id,
+            'Submission when instance question is closed',
+            '',
+            false,
+            false,
+            jsonb_build_object('grading_job_id', grading_job_id),
+            '{}'::jsonb,
```

Comment:
```suggestion
            v.id,
            'Submission when instance question is closed', -- student messsage
            '', -- instructor message
            false, -- manually reported
            false, -- course-caused
            jsonb_build_object('grading_job_id', grading_job_id), -- course data
            '{}'::jsonb, -- system data
```

Reply:
I should add that Matt and I discussed removing this sproc call entirely (we should be catching and handling this case far upstream from here), so to me this seems like overkill for code that we intend to rip out shortly. Of course, you didn't know that before commenting. I'll accept this so we can get this merged.